tests/console/dpdk.pm

```diff
@@ -23,10 +23,16 @@ use warnings;
 use testapi;
 use serial_terminal 'select_serial_terminal';
 use utils;
+use version_utils qw(is_sle is_leap is_tumbleweed is_leap is_opensuse);
 
 
 sub install_ovs_dpdk {
-    zypper_call('in openvswitch3 dpdk22 dpdk22-tools', timeout => 300);
+    if (is_opensuse) {
+        zypper_call('in openvswitch dpdk dpdk-tools', timeout => 300);
+    }
+    else {
+        zypper_call('in openvswitch3 dpdk22 dpdk22-tools', timeout => 300);
+    }
```

Comment:
if (is_sle('>=15-sp5') || is_leap('>=15.5')) {
       zypper_call('in openvswitch3 dpdk22 dpdk22-tools', timeout => 300);
}
else {
    zypper_call('in openvswitch dpdk dpdk-tools', timeout => 300);
}

Reply:
okay, thanks!